app/components/forgot-password.hbs

```diff
@@ -0,0 +1,18 @@
+<div class="container">
+
+    <h1 class="strips">NextGen</h1>
+
+
+    <label class="email">Email</label>
+    <input type="text" class="input" required>
+    <button class="button">Continue</button>
+
+
+    <p5 class="signup">Don't have an account?<LinkTo @route="signup">Sign up</LinkTo>
+    </p5>
+</div>
+<div class="terms">
```

Comment:
Better to use icons..instead of unicodes

Reply:
okay